modules/preview-react/menu/stories/examples/CustomMenuItem.tsx

```diff
@@ -7,7 +7,7 @@ export const CustomMenuItem = () => {
     <Menu title="Menu Title">
       <MenuItem>First Item</MenuItem>
       <MenuItem>Second Item</MenuItem>
-      <li role="menuItem" id="customMenu-3" tabIndex={-1} style={{listStyle: 'none'}}>
```

Comment:
I removed the `id` on the custom menu item since it's ignored (Menu overwrites the passed `id` with a `${menuId}-${index}` `id` when it clones the menu item [here](https://github.com/Workday/canvas-kit/blob/master/modules/preview-react/menu/lib/Menu.tsx#L145)).

Also, does that `title` need to be there for `Menu`? The old Menu stories didn't use `title` (see `stories.tsx` in the [Menu MDX PR](https://github.com/Workday/canvas-kit/pull/1203/files)), seems like `title` was added when creating the new examples in the MDX PR. I did notice `title` was used in [a code snippet](https://github.com/Workday/canvas-kit/pull/1203/files#diff-90044480b1e84345b67ce916bc763d6d48e617af129e933da7df24defd743458L34) in the old README, but that was the only place I saw it in the old Menu docs.

Reply:
I would say remove it if it's not recommended for a11y (it doesn't appear to be: https://www.w3.org/TR/wai-aria-1.1/#menu)

Reply:
Agreed. I don't think we're doing that on any of our menus.